src/admin-views/settings/tickets-commerce/gateways.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * TEmplate to display a list of gateway options.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
+ * @var array                        $gateways          Array of gateway objects.
+ */
+
+$manager = tribe( TEC\Tickets\Commerce\Gateways\Manager::class );
+
+$brand_info = [
+    'paypal' => [
+        'logo_url' =>  Tribe__Tickets__Main::instance()->plugin_url . 'src/resources/images/admin/paypal_logo.png',
+        'subtitle' => 'Enable payments through PayPal, Venmo, and credit card',
```

Comment:
Localization methods.

Reply:
Oh.... I actually hard-coded these for the demo. I'll move them.